coffee-rest/src/main/java/hu/icellmobilsoft/coffee/rest/exception/DefaultGeneralExceptionMapper.java

```diff
@@ -82,80 +86,108 @@
     public Response toResponse(Exception e) {
         Response result = null;
         if (e instanceof BaseExceptionWrapper<?>) {
-            BaseExceptionWrapper<?> wrappedException = (BaseExceptionWrapper<?>) e;
-            if (wrappedException.getException() != null) {
-                log.info("Wrapped exception. Trying to match the correct mapper...");
-                result = handleWrappedException(wrappedException.getException());
-            } else if (e.getCause() instanceof BaseException) {
-                log.info("Wrapped BaseException. Trying to match the correct mapper...");
-                result = handleWrappedException((BaseException) e.getCause());
-            } else {
-                log.error("Unknown error in cause: ", e);
-                log.writeLogToError();
+            Exception unwrappedException = unwrapException((Exception & BaseExceptionWrapper<?>) e);
+            if (unwrappedException instanceof BaseException) {
+                result = handleWrappedException((BaseException) unwrappedException);
```

Comment:
ez lehetne egybol return is, a result nem kell

Reply:
A result lehet null is; ilyenkor eddig is ment tovább és ráfutott a sima `handleException`.

Reply:
arra gondoltam hogy a `result` változóra nincs szükség